src/employee_management.c

```diff
@@ -11,6 +11,7 @@
 #define MIN_YEAR 2010
 #define MAX_MONTH 12
 #define MIN_MONTH 1
+#define sal_year 2020
```

Comment:
Can we consider using the system year instead ?

Reply:
That's a good idea, thanks for pointing out.